### ISSUE TITLE
pupulating dynamically does not work

### ISSUE THREAD

**Issue by widonwaker** [lyfeyaj/swipe#55]
```
var output = $('.swipe-wrap');
$.ajax({
        url: 'http://mywebsite.com/images/images_to_swiper.php',
        dataType: 'jsonp',
        jsonp: 'jsoncallback',
        timeout: 5000,
        success: function(data, status){
            $.each(data, function(i,item){ 
                var landmark = '<div><img src="http://mywebsite.com//images/'+item.img+'" style="width:100%;"></img></div>';           
                output.append(landmark);

            }); 

        },
        error: function(){
            $( "#popupDialog" ).popup( "open" )
        }
    });
```

my php page returns a json encoded string like this:

`([{"id":"19","img":"image1.png","posizione":"1"},{"id":"11","img":"anotherimage.jpg","posizione":"6"}]);`

but slider is not populated. Any idea why?

**Comment by lyfeyaj** [comment]
You should try to initialize swipe after your data fetched

**Comment by widonwaker** [comment]
Thank you for your answer, anyway have a look here: https://jsfiddle.net/ywn1w7fn/2/

wherever i put my ajax process to fecth data, it's totally ignored. No data appears in the swiper. I've put my scripts in this order:
1)Fetch data
2)swipe.js
3)initialize

**Comment by lyfeyaj** [comment]
Your code snippets in jsfiddle is working... as the following picture
![image](https://cloud.githubusercontent.com/assets/1445568/20881459/5e8ef6e8-bb17-11e6-866a-60373984ab7e.png)

You may arrange your code logic like this:

```js
var output = $('.swipe-wrap');
$.ajax({
        url: 'http://mywebsite.com/images/images_to_swiper.php',
        dataType: 'jsonp',
        jsonp: 'jsoncallback',
        timeout: 5000,
        success: function(data, status){
            $.each(data, function(i,item){ 
                var landmark = '<div><img src="http://mywebsite.com//images/'+item.img+'" style="width:100%;"></img></div>';           
                output.append(landmark);
            }); 

           // Initialize Swipe here
            new Swipe(document.getElementById('mySwipe'));
        },
        error: function(){
            $( "#popupDialog" ).popup( "open" )
        }
    });
```

**Comment by widonwaker** [comment]
not working, look: https://jsfiddle.net/ywn1w7fn/3/

i used an existing php file returning a json string (check here: http://coinfarm.eu/test.php) . :(

**Comment by lyfeyaj** [comment]
You are not include jquery, see your console error report 
![image](https://cloud.githubusercontent.com/assets/1445568/20882495/35e1fccc-bb1c-11e6-9977-6731b6aa5d1a.png)

And besides, Chrome will block your ajax request because jsfiddle is using `https` while your data source is a `http`

**Comment by widonwaker** [comment]
included, same result: http://jsfiddle.net/ywn1w7fn/5/

even with http. Tried with IE, Edge, Firefox, Chrome. Same result for all. I'm sorry if i'm annoying you..

**Comment by lyfeyaj** [comment]
Your ajax request has error reported, seems like your jsonp response is not correct.

![image](https://cloud.githubusercontent.com/assets/1445568/20883716/8d00cbcc-bb22-11e6-88c3-0c806c8c0bf4.png)

**Comment by widonwaker** [comment]
```
$sql = "SELECT * FROM homeslide ORDER BY posizione ASC";
$result = mysqli_query($con,$sql) or die ("Query error: " . mysqli_error());

if (mysqli_num_rows($result)<1) {
$sql = "select * from homeslide";
$result = mysqli_query($con,$sql) or die ("Query error: " . mysqli_error());
}

$records = array();

while($row = mysqli_fetch_assoc($result)) {
	$records[] = $row;
}

mysqli_close($con);

echo $_GET['jsoncallback'] . '(' . json_encode($records) . ');';
```

what's printed out:
```
<br />
<b>Notice</b>:  Undefined index: jsoncallback in <b>/home/rj1sfvw9/public_html/sscg/db_to_app_home.php</b> on line <b>29</b><br />
([{"id":"19","img":"img1.png","posizione":"1"},{"id":"18","img":"img2.png","posizione":"2"},{"id":"17","img":"another.png","posizione":"3"},{"id":"16","img":"thistoo.png","posizione":"4"},{"id":"15","img":"adapt.png","posizione":"5"},{"id":"11","img":"imager.jpg","posizione":"6"}]);
```

**Comment by widonwaker** [comment]
I found the solution: for unknown reasons, using a different jquery version of 1.9.1 as in this code demo, does not work. This script works only with jquery 1.9.1

**Comment by lyfeyaj** [comment]
So, your problem was solved by changing jquery version?

**Comment by lyfeyaj** [comment]
@widonwaker Is your problem solved?

**Comment by widonwaker** [comment]
yes, the problem is solved changing jquery version to 1.9.1 .
